jsapp/js/hooks/useApiFetcher.hook.ts

```diff
@@ -3,44 +3,70 @@ import {useCallback, useEffect, useState} from 'react';
 export interface ApiFetcherOptions {
   /* If set, the ApiFetcher will execute the fetch() function once per the
    * provided number of seconds. Will only fetch when the window is visible.
-   * *Don't* use this for fetches that mutate data on the server (POST, DELETE, PUT).
+   * *Don't* use this for fetches that mutate data on the server (DELETE, PUT, some POST requests).
```

Comment:
Is there a concise way to explain "some". I don't mean to suggest every comment become a tomb, but if there is a short word or two to replace "some" then it would be better.